templates/environment/versions/cf-v1.2.0.yml

```diff
@@ -0,0 +1,408 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Metadata:
+  Version: 'v1.1.0'
```

Comment:
Maybe we need v1.3.0 for this feature. And would you mind to give a diff for the new template vs the current one?